pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -482,11 +482,14 @@ def _binary_op_method_timedeltalike(op, name):
             # the PyDateTime_CheckExact case is for a datetime object that
             # is specifically *not* a Timestamp, as the Timestamp case will be
             # handled after `_validate_ops_compat` returns False below
-            from ..tslib import Timestamp
+            from timestamps import Timestamp
             return op(self, Timestamp(other))
             # We are implicitly requiring the canonical behavior to be
             # defined by Timestamp methods.
 
+        elif is_timedelta64_object(other):
+            return op(self, Timedelta(other))
```

Comment:
you can do the same here? (add the `and not is_timedelta64_object(other)` on the line below)

Reply:
How strong is your opinion on this one?  I have a slight preference for the explicitness of this ordering over the sometimes-vagueness of a catch-all.

Reply:
Not that strong, but the thing is that this will then again go through this method, instead of being run once. 
If you want to deal with it explicitly, you can also do something like

```
elif is_timedelta64_object(other):
    pass  # other coerced to Timedelta at the bottom
```